builders/build/eth-api/dev-env/brownie.md

```diff
@@ -144,15 +166,38 @@ If you would like to set a default network, you can do so by adding the followin
 !!! note
     Keep in mind that the `brownie-config.yaml` file isn't automatically created, you can optionally create it yourself.
 
+### Setting your Networks RPC URLs
+
+It is recommended that you override the default Brownie RPC URLs to [your own RPC endpoint](/builders/get-started/endpoints/#rpc-endpoint-providers){target=_blank} or the public Moonbeam network endpoints. You can override the default Brownie RPC URL for each network as follows:
+
+=== "Moonbeam"
+
+    ```bash
+    brownie networks modify moonbeam-main host={{ networks.moonbeam.rpc_url }}
+    ```
+
+=== "Moonriver"
+
+    ```bash
+    brownie networks modify moonriver-main host={{ networks.moonriver.rpc_url }}
+    ```
+
+=== "Moonbase Alpha"
+
+    ```bash
+    brownie networks modify moonbeam-test host={{ networks.moonbase.rpc_url }}
+    ```
+
+
```

Comment:
I'm going to include a termynal snippet here, once that's merged in. Not including a screenshot here so that we don't have to then remove it / change the chinese docs site as well.